tools/ci_build/github/linux/docker/scripts/install_centos.sh

```diff
@@ -1,12 +1,7 @@
 #!/bin/bash
 set -e
 
-yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm https://centos7.iuscommunity.org/ius-release.rpm
+yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 
-yum remove --tolerant cmake git
-yum install -y cmake3 git2u-all
-ln -s /usr/bin/cmake3 /usr/bin/cmake
-ln -s /usr/bin/ctest3 /usr/bin/ctest
-
-yum install -y redhat-lsb-core expat-devel libcurl-devel protobuf-compiler protobuf-devel protobuf rpmdevtools tar unzip ccache curl gcc gcc-c++ zlib-devel make  python2-devel python3-devel python3-numpy libunwind icu aria2 rsync python3-setuptools python3-wheel bzip2
+yum install -y redhat-lsb-core expat-devel libcurl-devel rpmdevtools tar unzip ccache curl gcc gcc-c++ zlib-devel make  python2-devel python3-devel python3-pip python3-numpy libunwind icu aria2 rsync python3-setuptools python3-wheel bzip2 git
```

Comment:
> git [](start = 238, length = 4)

We don't want this git. It is too old

Reply:
But it's good enough to running the build.

Reply:
But it is NOT good enough if we want to login and do rudimentary things.

---
In reply to: [331164812](https://github.com/microsoft/onnxruntime/pull/1986#discussion_r331164812) [](ancestors = 331164812)

Reply:
Then, may I build the git from source? I don't quite trust third-party yum repos.

Reply:
And how does it fix it, building from their source?

---
In reply to: [331166421](https://github.com/microsoft/onnxruntime/pull/1986#discussion_r331166421) [](ancestors = 331166421)